tests/integration/test_filter.py

```diff
@@ -253,3 +253,35 @@ def test_filtered_csv(tmpdir, c):
     expected_df = df[df["b"] < 10]
 
     assert_eq(return_df, expected_df)
+
+
+@pytest.mark.gpu
+def test_filter_decimal(c):
+    import cudf
+
+    df = cudf.DataFrame(
+        {
+            "a": [304.5, 35.305, 9.043, 102.424, 53.34],
+            "b": [2.2, 82.4, 42, 76.9, 54.4],
+            "c": [1, 2, 2, 5, 9],
+        }
+    )
+    df["a"] = df["a"].astype(cudf.Decimal64Dtype(12, 3))
+    df["b"] = df["b"].astype(cudf.Decimal64Dtype(7, 1))
+    c.create_table("df", df)
+
+    result_df = c.sql(
+        """
+        SELECT
+            c
+        FROM
+            df
+        WHERE
+            a < b
```

Comment:
This is filtering by columns, but it might be worthwhile to either add a test or modify this one to filter by a scalar as well, to ensure that we have handling for creating a decimal scalar.

Reply:
Getting a cudf error when filtering by a scalar that I'm investigating